server/routes/routerRequestHandlers/checkReviewPlanDtoExistsInPrisonerContext.ts

```diff
@@ -6,14 +6,22 @@ import { getPrisonerContext } from '../../data/session/prisonerContexts'
  * Request handler function to check the ReviewDto exists in the prisoner context.
  */
 const checkReviewPlanDtoExistsInPrisonerContext = async (req: Request, res: Response, next: NextFunction) => {
-  // TODO - replace with DTO check once implemented
   const urlPath = req.originalUrl || ''
+  const prisonerContext = getPrisonerContext(req.session, req.params.prisonNumber)
 
+  // Exemption-specific check
   if (urlPath.includes('/review/exemption')) {
+    if (!prisonerContext.reviewExemptionDto) {
```

Comment:
This appears to be doing what the previous request handler is doing (though I dont think the previous request handler is being called anywhere now 🤔 )

I think what you want to do here is call/delegate to the other request handler:
```
  // Exemption-specific check
  if (urlPath.includes('/review/exemption')) {
    return checkReviewExemptionDtoExistsInPrisonerContext(req, res, next)
  }
```

I'll be honest, I'm not a massive fan of this approach/pattern; but for the time being lets go with it
(personally I'd like clear route definitions calling their relevant route request handlers, and not really keen on one handler calling another ... 😬 . But lets go with it for now, we can always come back to it as we iterate and implement further code)